rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -112,6 +112,8 @@ void TensorInfo::reset_tensor_roi_buffers() {
         size_t roi_size = (_layout == RocalTensorlayout::NFCHW || _layout == RocalTensorlayout::NFHWC) ? _dims[0] * _dims[1] : _batch_size; // For Sequences pre allocating the ROI to N * F to replicate in OpenVX extensions
         allocate_host_or_pinned_mem((void **)&_roi_buf, roi_size * 4 * sizeof(unsigned), _mem_type);
     }
+    if(!_original_roi_height->size()) _original_roi_height = std::make_shared<std::vector<uint32_t>>(_batch_size);
```

Comment:
Why is changed from struct roi to width and height

Reply:
Removed these _original_roi_width and _original_roi_height as they are not used anymore